components/Collection/Collection.js

```diff
@@ -72,67 +81,119 @@ export default function Collection({
               }
               id={id}
               variant="correct"
+              collectionTitle={true}
               actionMode={actionMode}
               flashcardSelection={flashcardSelection}
               changeFlashcardSelection={changeFlashcardSelection}
             />
-          </IconWrapper>
+          </IconCorrectWrapper>
         </CollectionBox>
       )}
-    </>
+    </CollectionBoxWrapper>
   );
 }
 
+const CollectionBoxWrapper = styled.div`
+  margin: 0 auto;
+  width: 90vw;
+  height: 218px;
+  max-width: 500px;
+  position: relative;
+
+  @media (min-width: 768px) {
+    min-height: 300px;
+    max-height: 400px;
+  }
+`;
+
+const CollectionBoxShadow = styled.div`
+  margin: 0 auto;
+  width: 90vw;
+  height: 218px;
+  max-width: 500px;
+  position: absolute;
+  border-radius: 10px;
+
+  @media (min-width: 768px) {
+    min-height: 300px;
+    max-height: 400px;
+  }
+`;
+
+const CollectionBoxShadow1 = styled(CollectionBoxShadow)`
+  top: 8px;
+  left: 8px;
+  background-color: var(--secondary-light-grey);
+`;
+
+const CollectionBoxShadow2 = styled(CollectionBoxShadow)`
+  top: 16px;
+  left: -8px;
+  background-color: var(--secondary-grey);
+`;
+
 const CollectionBox = styled(Link)`
   margin: 0 auto;
-  width: 20rem;
-  height: 13rem;
+  width: 90vw;
+  height: 218px;
+  max-width: 500px;
   position: relative;
   border-radius: 10px;
   border: 2px solid ${({ $color }) => $color};
-  padding: 10px;
+  display: grid;
+  grid-template-columns: var(--grid-columns-card-and-title);
+  grid-template-rows: var(--grid-rows-card-and-title);
   background-color: #fff;
   text-decoration: none;
   color: #000;
 
-  &::before,
-  &::after {
-    content: "";
-    position: absolute;
-    width: 100%;
-    height: 100%;
-    border-radius: 10px;
-    border: 2px solid ${({ $color }) => $color};
-    z-index: -1;
-  }
-
-  &::after {
-    top: 10px;
-    right: 10px;
-    background-color: #ddd;
-  }
-
-  &::before {
-    top: 19px;
-    left: 10px;
-    background-color: #bbb;
+  @media (min-width: 768px) {
+    min-height: 300px;
+    max-height: 400px;
   }
 `;
 
 const CollectionName = styled.h3`
-  font-size: 1.5rem;
-  padding-top: 50px;
+  font: var(--collection-name);
   text-align: center;
+  align-self: center;
+  grid-column: 1 / 8;
+  grid-row: 2 / 3;
 `;
 
 const IconWrapper = styled.section`
```

Comment:
If this element only wraps your icons, which are buttons, and is used to align them, you would not use a section element, but a div element.

A section element is typically characterized by containing a heading.

Reply:
Hi Niklas, thanks for your hint we have changed the ```styled.section``` into a ```styled.div```. 🙃